### PR TITLE
Polish wizard

### PR DESCRIPTION
**Pull request by fmoc**
WIP, more changes to come.

screenshot_2022-08-16_15-24-35
screenshot_2022-08-16_15-37-54

### CODE REVIEW

**Comment by TheOneRing** [comment]
🇵🇱

File: src/gui/folderman.cpp [hunk 1]
```diff
@@ -1220,7 +1220,28 @@ QString FolderMan::trayTooltipStatusString(
     return folderMessage;
 }
 
-static QString checkPathValidityRecursive(const QString &path)
+// QFileInfo::canonicalPath returns an empty string if the file does not exist.
+// This function also works with files that does not exist and resolve the symlinks in the
+// parent directories.
+static QString canonicalPath(const QString &path)
```
Comment (TheOneRing) [inline src/gui/folderman.cpp:1226]: maybe move it to anonymous namespace?

Comment (fmoc) [reply src/gui/folderman.cpp:1226]: I have not touched this method, just moved `checkPathValidityRecursive` below it (to have class methods next to each other).

**Comment by fmoc** [comment]
screenshot_2022-08-17_16-02-10

**Review by TheOneRing — APPROVED** [review]

**Pull request merged**